youtube_dl/extractor/vxxx.py

```diff
@@ -0,0 +1,240 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import base64
+
+from .common import InfoExtractor
+from ..utils import (
+    float_or_none,
+    int_or_none,
+    parse_duration,
+    strip_or_none,
+    unified_timestamp,
+    url_or_none,
+)
+
```

Comment:
`str.maketrans()` and `str.translate()` need to be shimmed for Python 2.

If this works the definitions can eventually be moved into `compat.py`.
```suggestion
try:
    compat_str_maketrans, compat_str_translate = (
        compat_str.maketrans,
        lambda s, table: s.translate(table)
    )
except AttributeError:
    # Python 2
    def compat_str_maketrans(x, *args):
        if not args:
            return x
        y, z = args[0], args[1] if len(args) > 1 else ''
        if len(x) != len(y):
            raise ValueError(
                'the first two maketrans arguments must have equal length')
        tbl = dict(zip(x, y))
        tbl.update((k, None) for k in z)
        return tbl

    def compat_str_translate(s, table):
        def xlate(c):
            try:
                return table[c] or ''
            except LookupError:
                return c
        return ''.join(xlate(c) for c in s)

```